tele_giphy/game/templates/game/pre_game_room.html

```diff
@@ -43,15 +43,14 @@
         {% if request.session.game_mode == 'multiplayer' %}
           <div class="row">
             <div class="col-md-4">
-              <h4>Current amount of players: 5 (PLACEHOLDER VALUE)</h4>
+              <h4>Current amount of players: {{ users.count }}</h4>
             </div>
             <div class="col-md-4 col-md-offset-4 text-right">
-              <h4>List of players joined: (ALL PLACEHOLDER VALUES)</h4>
+              <h4>List of players joined:</h4>
```

Comment:
Yeah sorry I keep on thinking of the design stuff of things too since that's where I've had most experience when it comes to websites heh